types/source/dotnet/csproj.go

```diff
@@ -0,0 +1,133 @@
+/*
+ *  Copyright IBM Corporation 2021
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package dotnet
+
+import (
+	"encoding/xml"
+	"io"
+	"regexp"
+)
+
+var (
+	// FourXPattern is the framework version pattern
+	FourXPattern = regexp.MustCompile("v4*")
+
+	// WebLib is the key library used in web applications
+	WebLib = regexp.MustCompile("^System.Web*")
```

Comment:
This also seems to be not the intended regex. It is looking for 0 or more b's in the end